src/redux/configureStore.js

```diff
@@ -0,0 +1,18 @@
+import ActionType from './categorie/categories';
+
+const initialState = [];
+
+const booksReducer = (state = initialState, action) => {
+  switch (action.type) {
+    case ActionType.ADD_BOOK:
+      return [...state, action.payload];
+
+    case ActionType.REMOVE_BOOK:
+      return state.filter((book) => book.id !== action.payload);
+
+    default:
+      return state;
+  }
+};
+
```

Comment:
- [ ] As the previous reviewer mentioned, please manage to combine both of the books and categories reducers into the redux store using the react-redux giving methods. 🙏

Reply:
Thank you for your feedback.